components/raftstore/src/store/worker/pd.rs

```diff
@@ -1786,6 +1831,57 @@ where
             health_service.set_serving_status("", status);
         }
     }
+
+    fn is_store_heartbeat_delayed(&self) -> bool {
+        let now = UnixSecs::now();
+        let interval_second = now.into_inner() - self.store_stat.last_report_ts.into_inner();
+        // To avoid reporting too many fake heartbeats to PD, we should also
```

Comment:
Too many comments in code for the logic `fake heartbeats`, any chance we could make it formal code?